src/validator/client/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Fetcher } from "./fetcher";
+import { ApiResponse, ApiError, FetchConfig } from "./types";
```

Comment:
Need to add `.js` extension to imports here:
 
```
import { Fetcher } from "./fetcher.js";
import { ApiResponse, ApiError, FetchConfig } from "./types.js";
```

Reply:
Was this causing issues for you? If not, these come from an upstream lib and I'd like to avoid modifying the code too much if we can avoid it. If it is causing issues, then that's another thing and we'll have to do that.

Reply:
Yes, unfortunately it was causing issues.

Reply:
Bummer, ok!